android-gif-drawable/src/main/java/pl/droidsonroids/gif/GifDrawable.java

```diff
@@ -1025,4 +1025,12 @@ public void setTransform(@Nullable Transform transform) {
     public Transform getTransform() {
         return mTransform;
     }
+
+    private boolean checkMultipleFrames() {
+        return getNumberOfFrames() > 1;
+    }
+
+    private boolean getIsRunning() {
```

Comment:
It makes no sense to add yet another layer of abstraction here. It only hinders readability.
But you can call one of the existing methods from the other one.
Perhaps one from Animatable should be considered secondary.